code/datums/materials/alloys.dm

```diff
@@ -0,0 +1,124 @@
+/** Materials made from other materials.
+  */
+/datum/material/alloy
```

Comment:
This type is now included in any places that does subtypesof(), I don't think thats a lot of places but I do know SSmaterials will initialize this material now, can you make sure that that doesn't happen? :)

Reply:
Should have been fixed, though you should probably give it another look.